pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -393,6 +429,8 @@ func GetBootstrapClusterRoles() []authorizationapi.ClusterRole {
 		{
 			ObjectMeta: kapi.ObjectMeta{
 				Name: BasicUserRoleName,
+				Annotations: map[string]string{
+					oapi.OpenShiftDescription: "A user that can get basic information about projects."},
```

Comment:
So we have to be vague enough that the description doesn't get too far out of sync since the specifics of rules will almost certainly change over time.  Right now it means:

1. Get info about yourself
2. List `projectrequests`
3. Get/List `clusterroles`
4. List `storageclasses`
5. List/Watch `projects`